System/RTETools.cpp

```diff
@@ -124,8 +124,8 @@ namespace RTE {
 
 	bool ASCIIFileContainsString(std::string filePath, std::string findString) {
```

Comment:
It's not specific for ini files though. It works for probably anything that can contain text.
I tested on a .txt file, then tested on the same file with the extension removed (so it's just `file`) and that works too.
This method was originally used to read the license key and my plan was to repurpose it for achievements/unlockables.

Reply:
Yeah, I kinda put emphasis on the wrong thing there. My problem is that it supports a specific comment format but nothing in that method definition suggests it even knows or cares about block comments. Don't have to fix it in this PR (can just put a TODO), but I think it'd be better with a boolean that turns on/off caring about comments, since that adds functionality and makes it clear that it'll normally care about comments.
Also, maybe I'm mis-remembering now but does it only care about block comments and not // comments? If so, that's weird too and should be mentioned in a TODO or whatever.

Reply:
I think in the grand scheme it doesn't matter, and we'll probably be removing the part about comments all together.
The only thing this will be looking for are hash values in a file that has nothing except hash values in it.

Reply:
Yeah sure, that's fine then, leave it be for now. If you're gonna rework it later and it's not being used at the moment, then it doesn't matter hugely if it's a little weird for now.